File: 75-sort-colors/sort-colors.java
class Solution {
    public void swap(int[] nums,int a,int b){
        int temp=nums[a];
        nums[a]=nums[b];
        nums[b]=temp;
    }
    public void sortColors(int[] nums) {
        int left=0;
        int right=nums.length-1;
        int index=0;
        while(index<=right){
            if(nums[index]==0){
                swap(nums,left++,index++);
            }else if(nums[index]==2){
                swap(nums,right--,index);
            }else{
                index++;
            }
        }
    }
}

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        left,right,index=0,len(nums)-1,0
        while index<=right:
            if nums[index]==0:
                nums[left],nums[index]=nums[index],nums[left]
                left+=1
                index+=1
            elif nums[index]==2:
                nums[right],nums[index]=nums[index],nums[right]
                right-=1
            else:
                index+=1
        